src/breakpoints/index.ts

```diff
@@ -52,13 +67,76 @@ class BreakpointsHeader extends Widget {
 }
 
 export namespace Breakpoints {
+  export interface IBreakpoint extends DebugProtocol.Breakpoint {}
+
   /**
    * The breakpoints UI model.
    */
   export interface IModel {}
 
+  export class IModel implements IModel {
+    constructor(model: IBreakpoint[]) {
+      this._state = model;
+    }
+
+    get breakpointsChanged(): ISignal<this, IBreakpoint[]> {
+      return this._breakpointsChanged;
+    }
+
+    get activesChange(): ISignal<this, boolean> {
```

Comment:
To follow the same naming convention as for the other signals:

```suggestion
    get activeChanged(): ISignal<this, boolean> {
```